src/ai/backend/common/validators.py

```diff
@@ -465,12 +465,14 @@ def check_and_return(self, value: Any) -> datetime.timedelta:
 
 class Slug(t.Trafaret, metaclass=StringLengthMeta):
 
-    _rx_slug = re.compile(r'^[a-zA-Z0-9]([a-zA-Z0-9._-]*[a-zA-Z0-9])?$')
+    _regexp = r'^((?!.*?\.{2,}))[\w.\-\/\\\s]+$'
+    _rx_slug = t.RegexpRaw(regexp=_regexp)  # re.compile(r'^[\w\-.\s]+$')
```

Comment:
Could you explain what's the intended change here?
Doesn't it interfere with `allow_dot` option or isn't it over-complication due to it?
Also remove the comment if you're sure that the new code is better and includes all the behavior of the old code.

Reply:
Hmm, I thought some users could write folder name such as ```./foo```, which is as the same meaning as ```foo```.
But, As your comments above, I guess I overthought from the real purpose for the regular expression about vfolder.
I'll rollback the regex, letting ```allow_dot``` member variable do its job.